app/sw.js

```diff
@@ -1,51 +1,62 @@
-/* globals self,caches,importScripts,RangedResponse,toolbox */
+/* globals self,caches,RangedResponse,toolbox */
 
-const OFFLINE_VIDEO_REGEX = /cdn\.jwplayer\.com\/videos\/(.)+\.mp4$/;
+/* inject:vendorscripts */
 
-importScripts('sw-toolbox.js');
-importScripts('ranged-request.js');
+var OFFLINE_VIDEO_REGEX = /cdn\.jwplayer\.com\/videos\/(.)+\.mp4$/;
 
 toolbox.options.debug      = false;
-toolbox.options.cache.name = 'jw-showcase-v3.2.0';
+toolbox.options.cache.name = 'jw-showcase';
 
 toolbox.precache([
     '/',
-    'config.json',
-    'manifest.json',
-    'favicon.ico',
-    'styles/main.css',
-    'fonts/icons.eot',
-    'fonts/icons.svg',
-    'fonts/icons.ttf',
-    'fonts/icons.woff',
-    /* inject:scripts */
+    '/index.html',
```

Comment:
Do absolute paths work when Showcase is hosted in a subdirectory?

Reply:
For now, priority was given to support PWA in hosted Showcases. It's uncertain when (or if) PWA support is going to be broadened to self-hosted/open source.